### DISCUSSION TITLE
[TimeInput] Does it   support controlled mode with  value and onChange ?

### DISCUSSION THREAD

**Discussion by vaynevayne** [mantinedev/mantine#3834] [Q&A]
I found no information about the controlled mode in either the documentation or the source code. Is it not currently supported? When will it be supported?

https://mantine.dev/dates/time-input/

**Comment by yverby** [comment]
`TimeInput` component is a pure input `input[type='time']` with [Input](https://mantine.dev/core/input/) component styles
You can make it controllable, just use `value` and `onChange` props

**Reply by vaynevayne** [reply]
Well, that's true, but the native input is bound to onInput events and needs to be converted, so I think manitine still needs to provide the controlled mode and add it to the documentation
